changelog/_unreleased/2021-04-08-sidebar-flash-on-cookie-accept-all.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Sidebar flash on cookie accept all
+issue: /
+author: Rune Laenen
+author_email: rune@laenen.me 
+author_github: runelaenen
+---
+# Storefront
+*  Changed `cookie-configuration.plugin.js` to not flash the cookie sidebar when Accept All is clicked from the cookie bar.
```

Comment:
Please go a bit more into detail here, for example: which methods have been removed, deprecated, changed etc.